docs/deployment/getting_started_locally.md

```diff
@@ -97,7 +97,9 @@ Afterwards, the Gardener resources will be deployed into the cluster.
 You can wait for the `Seed` to be ready by running:
 
 ```bash
-kubectl wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed local --timeout=5m
+kubectl wait --for=condition=gardenletready seed local --timeout=5m
+kubectl wait --for=condition=extensionsready seed local --timeout=5m
+kubectl wait --for=condition=bootstrapped seed local --timeout=5m
```

Comment:
What about all these - shouldn't they be adapted as well just like in `skaffold.yaml`? 

Also, there are similar statements in https://github.com/gardener/gardener/blob/master/docs/development/getting_started_locally.md (maybe more, please check).